maqs-webservices/src/test/java/com/cognizantsoftvision/maqs/webservices/WebServiceDriverUnitTest.java

```diff
@@ -12,7 +12,7 @@
 /**
  * The Web Service Driver unit test class.
  */
-public class WebServiceDriverUnitTest {
+public class WebServiceDriverUnitTest extends BaseWebServiceTest {
```

Comment:
Why are we extending base service if it is not required for the tests?

Reply:
changed back to baseGenericTest